website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+|             **example** | _CA/CA/UniqueId1_                                                                   |
+|               **notes** | -                                                                                   |
+|              **status** | Stable                                                                              |
+
+### 3. Given Name
+
+|          attribute name | Given Name                           |
+| ----------------------: | :----------------------------------- |
+|         **description** | The user's first name                |
+|   **SAML Attribute(s)** | `urn:oid:2.5.4.42` (givenName)       |
+|          **OIDC scope** | `profile`                            |
+|       **OIDC claim(s)** | `given_name`                         |
+| **OIDC claim location** | Userinfo endpoint                    |
+|              **origin** | Provided by user's Identity Provider |
+|             **changes** | Yes                                  |
+|        **multiplicity** | Single-valued                        |
+|        **availability** | Always                               |
+|             **example** | _John_                               |
+|               **notes** | -                                    |
+|              **status** | Stable                               |
+
+### 4. Family Name
+
+|          attribute name | Family Name                          |
+| ----------------------: | :----------------------------------- |
+|         **description** | The user's last name                 |
+|   **SAML Attribute(s)** | `urn:oid:2.5.4.4` (sn)               |
+|          **OIDC scope** | `profile`                            |
+|       **OIDC claim(s)** | `family_name`                        |
+| **OIDC claim location** | Userinfo endpoint                    |
+|              **origin** | Provided by user's Identity Provider |
+|             **changes** | Yes                                  |
+|        **multiplicity** | Single-valued                        |
+|        **availability** | Always                               |
+|             **example** | _Doe_                                |
+|               **notes** | -                                    |
+|              **status** | Stable                               |
+
+### 5. Birth Date
+
+|          attribute name | Birth Date                                           |
+| ----------------------: | :--------------------------------------------------- |
+|         **description** | The user's birth date                                |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.25178.1.2.3` (schacDateOfBirth) |
+|          **OIDC scope** | `profile`                                            |
+|       **OIDC claim(s)** | `birth_date`                                         |
+| **OIDC claim location** | Userinfo endpoint                                    |
+|              **origin** | Provided by user's Identity Provider                 |
```

Comment:
Question: Is this mapped from the eIDAS `DateOfBirth` attribute in the case of natural person authentication through eIDAS?